docker-compose-full-cm.yml

```diff
@@ -69,8 +69,13 @@ services:
         MAPBOX_INIT_ZOOM: ${MAPBOX_INIT_ZOOM}
         CVIZ_API_SERVER_URL: ${CVIZ_API_SERVER_URL}
         CVIZ_API_WS_URL: ${CVIZ_API_WS_URL}
+        WEBAPP_THEME_LIGHT: ${WEBAPP_THEME_LIGHT}
```

Comment:
Please make a pull request in CDOT DevOps for new environment variables if these are required so we don't break the CDOT deployment once this is merged.

Reply:
These have been added to the devops pipeline, and there is an active PR to include the cdot icon in webapp deployments as a volume: https://dev.azure.com/SOC-OIT/CDOT/_git/cdot-cv-gcp/pullrequest/85984